cmd/tailscale/ui.go

```diff
@@ -839,12 +839,22 @@ func (ui *UI) layoutPeer(gtx layout.Context, sysIns system.Insets, p *UIPeer, us
 				}),
 				layout.Rigid(func(gtx C) D {
 					var addrs []string
+					var ip4, ip6 string
 					for _, addr := range p.Peer.Addresses {
 						if addr.IP.Is6() {
-							// Don't surface IPv6 addresses.
-							continue
+							if ip6 == "" {
+								ip6 = addr.IP.String()
+							}
+						} else {
+							if ip4 == "" {
+								ip4 = addr.IP.String()
```

Comment:
This code will select the last address for display. I don't know whether the ordering of Peer.Addresses is significant for that to matter.

Reply:
I think it selects the first address, but regardless: if we ever do support multiple IPv4 or multiple IPv6 addresses on a Tailscale interface, we'll need to consider which to display. Right now the first one will generally be the best one to display.